### ISSUE TITLE
Feature-Wish: option to MERGE databases / cache.db's

### ISSUE THREAD

**Issue by klartext** [akrennmair/newsbeuter#373]
If I have more than one newsbeuter accessing feeds, and I want to merge the databases,
an option to help with that would be very helpful.

**Comment by Minoru** [comment]
Is your use-case the same as in #175?

**Comment by klartext** [comment]
No.
I mean merge two cache-files into a cache-file that has all entries of the merged databeses now in one file.
But maybe export and import can be used as a way to merge databases.

**Comment by Minoru** [comment]
Yeah, I got what you want to achieve; I wonder *why* you need such a feature.

**Comment by klartext** [comment]
One newsbeuter at home, another one somewhere on a server, to fetch permanently some feeds that change to often for my personal intervals (and when my computer is not running).
Then from time to time importing the data from the newsbeuter on the server.

**Comment by Minoru** [comment]
The main problem I see with this is the fact that cache.db go hand in hand with `urls` file. In fact, Newsbeuter only displays feeds that are listed in `urls`. So after importing the feeds from the server, you'll have to add them to `urls` file to see them in Newsbeuter. That is both unwieldy and totally against your initial goal of separating your feeds into "private" and "server" sets.
